tests/test_auth.py

```diff
@@ -19,54 +19,71 @@ def setUp(self):
         self.login_route = 'api/v1/auth/login'
 
     def test_registration_conflict(self):
+        """ test registration conflict """
         req = self.client().post(self.register_route, data=self.user) # pylint: disable=unused-variable
         req1 = self.client().post(self.register_route, data=self.user) # pylint: disable=unused-variable
         self.assertEqual(req1.status_code, 409)
+        self.assertIn('exists', str(req1.data))
+        
     
     def test_registration_success(self):
+        """ test registration success """
         self.register = {"username":"spinderman", "password":"spinderman", "email":"spinderman@gmail.com"}
         req = self.client().post(self.register_route, data=self.register) # pylint: disable=unused-variable
         self.assertEqual(req.status_code, 201)
+        self.assertIn('success', str(req.data))
 
     def test_login_success(self):
+        """ test login success """
         self.user_data = {"email":"thor@gmail.com", "password":"thor"}
```

Comment:
I do not see where this person was registered before they could login. So how is this person able to login?

Reply:
let me fix that right now